test/lib/runtime/questionnaire_simulator_test.exs

```diff
@@ -442,6 +442,78 @@ defmodule QuestionnaireSimulatorTest do
     end
   end
 
+  defp ivr_questionnaire_with_steps(steps) do
+    questionnaire_with_steps(steps)
+    |> Questionnaire.changeset(%{modes: ["ivr"]})
+    |> Repo.update!()
+  end
+
+  describe "IVR - timeout message" do
```

Comment:
These tests make lots of sense to me. Nicely done! Also, reading them made me think we have some technical debt testing the whole IVR simulator. These new tests are the only ones testing it, right? Do we have an open issue for solving this debt?

Reply:
There are enough tests, at least for the IVR and SMS simulators, but they're _very_ repetitive. We could extract some helper methods and heavily simplify each test, improving the readability!

Reply:
I don't see them. For IVR, I only see the tests you wrote in this PR (all of them covering the `timeout` cases). Please help me: what tests am I missing?

Reply:
Oh, you're right, we're barely testing the IVR mode! We're not even testing the audio prompts (argh). I created #2074!